src/edgepi/peripherals/pwm.py

```diff
@@ -74,33 +75,36 @@ def set_duty_cycle_pwm(self, duty_cycle: float = None):
 
     def get_duty_cycle_pwm(self):
         """
-        get pwm duty cycle
+        Get pwm duty cycle
         Args:
             N/A
         Return:
             duty_cycle (float): duty cycle value from 0 to 1.0
         """
         return self.pwm.duty_cycle
 
-    def set_polarity_pwm(self, polarity: str = None):
+    def set_polarity_pwm(self, polarity: Polarity = None):
         """
-        set pwm polarity
+        Set pwm polarity
         Args:
-            polarity (str): "Normal" or "Inversed"
+            polarity (Polarity)
         Return:
             N/A
         """
-        self.pwm.polarity = polarity
+        if polarity == Polarity.NORMAL:
+            self.pwm.polarity = "normal"
+        elif polarity == Polarity.INVERSED:
+            self.pwm.polarity = "inversed"
 
     def get_polarity_pwm(self):
         """
-        get pwm polarity
+        Get pwm polarity
         Args:
             N/A
         Return:
-            polarity (str): "Normal" or "Inversed"
+            polarity (Polarity)
         """
-        return self.pwm.polarity
+        return Polarity[self.pwm.polarity.upper()]
```

Comment:
I think this is fragile. do explicit if elfi else.

Reply:
Done. https://github.com/EdgePi-Cloud/edgepi-python-sdk/pull/330/commits/19b1b9b77905dac49c2241f6f08694f5f38efe71